File: Android/doraemonkit-no-op/src/main/java/com/didichuxing/doraemonkit/kit/network/okhttp/DoraemonInterceptor.java
/*
 * Copyright (c) 2014-present, Facebook, Inc.
 * All rights reserved.
 *
 * This source code is licensed under the BSD-style license found in the
 * LICENSE file in the root directory of this source tree. An additional grant
 * of patent rights can be found in the PATENTS file in the same directory.
 */

package com.didichuxing.doraemonkit.kit.network.okhttp;


import java.io.IOException;

import okhttp3.Interceptor;
import okhttp3.Request;
import okhttp3.Response;

/**
 * 拦截器
 */
public class DoraemonInterceptor implements Interceptor {

    @Override
    public Response intercept(Chain chain) throws IOException {
            Request request = chain.request();
            return chain.proceed(request);
    }


}

File: Android/kotlin/doraemonkit-no-op/src/main/java/com/didichuxing/doraemonkit/kit/network/okhttp/interceptor/DoraemonInterceptor.kt
package com.didichuxing.doraemonkit.kit.network.okhttp.interceptor

import okhttp3.Interceptor
import okhttp3.Response
import java.io.IOException

/**
 * 抓包拦截器
 */
class DoraemonInterceptor : Interceptor {
    override fun intercept(chain: Interceptor.Chain): Response {
        val request = chain.request()
        return chain.proceed(request)
    }
}